test/Serializer/ArticleVoRNormalizerTest.php

```diff
@@ -471,34 +418,42 @@ public function normalizeProvider() : array
                     ],
                 ],
                 function (ApiTestCase $test) {
-                    $test->mockArticleCall('article1', true, true);
+                    $test->mockArticleCall('09560', true, true);
                 },
             ],
             'minimum snippet' => [
-                new ArticleVoR('article1', 1, 'research-article', '10.7554/eLife1', 'Author et al', null,
-                    'Article 1 title', $date, $statusDate, 1, 'e1', null, new ArraySequence([]), [], promise_for(null),
-                    promise_for(null), promise_for(new Copyright('CC-BY-4.0', 'Statement', 'Author et al')),
-                    new ArraySequence([new PersonAuthor(new PersonDetails('Author', 'Author'))]), null,
-                    promise_for(null), null, new ArraySequence([]), promise_for(null), new ArraySequence([
-                        new Section('Article article1 section title', 'articlearticle1section', [new Paragraph('Article article1 text')]),
-                    ]), new ArraySequence([]), new ArraySequence([]), new ArraySequence([]), new ArraySequence([]), promise_for(null),
-                    new ArraySequence([]), promise_for(null)),
+                Builder::for(ArticleVoR::class)
```

Comment:
looks like also these values come from defaults, as here only the missing values are specified explicitly. It may be worth it to add a few or all of the positive values that are reflected in the serialization